drivers/i2c/bme280_driver.go

```diff
@@ -113,6 +114,7 @@ func (d *BME280Driver) initHumidity() (err error) {
 	d.hc.h5 = 0 + (int16(addrE6) << 4) | (int16(addrE5) >> 4)
 
 	d.connection.WriteByteData(bme280RegisterControlHumidity, 0x3F)
```

Comment:
At some point, we should revisit the 0x3F magic constant here:

    d.connection.WriteByteData(bme280RegisterControlHumidity, 0x3F)

which existed before my PR as well as the similar line in the bmp280_driver.go `initialization()` function:

    d.connection.WriteByteData(bmp280RegisterControl, 0x3F)

Being able to specify the oversampling rates for temperature and pressure in the BMP280 driver, as well as humidity in the BME280 driver, would eliminate these magic constants and also ensure that we are using the correct values.